sdk/ml/azure-ai-ml/azure/ai/ml/operations/_workspace_operations.py

```diff
@@ -274,7 +280,34 @@ def begin_create(
             except HttpResponseError:
                 module_logger.warning("Failed to get parent hub for project, some values won't be transferred:")
         try:
-            return super().begin_create(workspace, update_dependent_resources=update_dependent_resources, **kwargs)
+            poller = super().begin_create(workspace, update_dependent_resources=update_dependent_resources, **kwargs)
+            # ai_services set - try to wait for workspace operation to complete, then make connection.
+            if ai_services is not None:
+                try:
+                    print(
```

Comment:
consider doing `logger.info(...)` instead of print

Reply:
Done

Reply:
Relevant code removed.